src/style.css

```diff
@@ -1,3 +1,68 @@
+@media (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  padding-left: 50px;
```

Comment:
Using fixed padding values like `padding-left: 50px;` might affect the layout on smaller screens. Consider using relative units or media queries to adjust padding for different screen sizes.